### PR TITLE
Remove Lombok

### PR DESCRIPTION
**Pull request by vpavic**
*Issue #, if available:*
See #189.

*Description of changes:*
This removes Lombok from the project.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/main/java/com/amazon/sqs/javamessaging/acknowledge/SQSMessageIdentifier.java [hunk 1]
```diff
@@ -63,4 +53,40 @@ public SQSMessageIdentifier(String queueUrl, String receiptHandle, String sqsMes
...
+                Objects.equals(queueUrl, that.queueUrl) && Objects.equals(receiptHandle, that.receiptHandle) && Objects.equals(sqsMessageId, that.sqsMessageId);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(queueUrl, receiptHandle, sqsMessageId);
+    }
+
+    @Override
+    public String toString() {
```
Comment (ziyanli-amazon) [inline src/main/java/com/amazon/sqs/javamessaging/acknowledge/SQSMessageIdentifier.java:88]: similar to the comment above

**Review by ziyanli-amazon — COMMENTED** [review]
Good job. Thanks for reducing the overheads for us. minor comment made

**Comment by vpavic** [comment]
Thanks for the feedback, I wasn't aware that `v1` branch didn't use Lombok.

I've aligned implementations of `#toString`, `#hashCode` and `#equals` as well as javadoc with `v1` branch.

**Review by ziyanli-amazon — APPROVED** [review]
Great! Thanks

**Pull request merged**